paasta_tools/marathon_serviceinit.py

```diff
@@ -138,22 +138,53 @@ def get_verbose_status_of_marathon_app(app):
     output.append("    App created: %s (%s)" % (str(create_datetime), humanize.naturaltime(create_datetime)))
     output.append("    Tasks:")
 
+    def get_datestring_for_date(date):
+        if date is None:
+            return 'None'
+        local_datetime = datetime_from_utc_to_local(date)
+        return '%s (%s)' % (
+            local_datetime.strftime("%Y-%m-%dT%H:%M"),
+            humanize.naturaltime(local_datetime),
+        )
+
     rows = [("Mesos Task ID", "Host deployed to", "Deployed at what localtime")]
     for task in app.tasks:
-        local_deployed_datetime = datetime_from_utc_to_local(task.staged_at)
         if task.host is not None:
             hostname = "%s:%s" % (task.host.split(".")[0], task.ports[0])
         else:
             hostname = "Unknown"
         rows.append((
             get_short_task_id(task.id),
             hostname,
-            '%s (%s)' % (
-                local_deployed_datetime.strftime("%Y-%m-%dT%H:%M"),
-                humanize.naturaltime(local_deployed_datetime),
-            )
+            get_datestring_for_date(task.staged_at),
         ))
     output.append('\n'.join(["      %s" % line for line in format_table(rows)]))
+
+    output.append("    Healthchecks:")
+
+    rows = [("Mesos Task ID", "Status", "First Success", "Last Success", "Last Failure",
```

Comment:
@solarkennedy are there any services that have more than one healthcheck? Marathon has health check results as an array, so I assure that's the most recent result for each health check.

Service authors might also care about Last Success.